web/src/containers/HomePage/HomePage.js

```diff
@@ -70,29 +71,42 @@ class HomePage extends React.Component {
     });
   }
 
+  enableSaveButton = () => {
+    setTimeout(()=> {
+      this.setState({
+        isButtonDisabled: false
+      });
+    }, 3000);
```

Comment:
Yeah, I have thought of it but for the edit Modal and all other forms. For the dashboard, the rows in are dynamic in the which it may get added by the user, which will increase the number of form inputs. So it will be though to manage local states.

Reply:
In that case also, I don't think its a good move to restrict the user for 3 seconds, because a situation may occur when the user wants to add more than 1 row, like he have 10 rows data and he wants to create a table of 10 rows first and then start filing the row with the data.

Reply:
in that case you can compare length of the rows to track dynamic rows